dashboard/app/models/pd/workshop_constants.rb

```diff
@@ -99,11 +99,5 @@ module WorkshopConstants
       COURSE_CSD => {course_name: 'csd-2019'},
       COURSE_CSP => {course_name: 'csp-2019'}
     }.freeze
-
```

Comment:
Moved this into `lib/cdo/shared_constants/pd/shared_workshop_constants.rb`, which is accessible to the script that takes some of our Ruby constants and makes them accessible in JS.